context-free.md

```diff
@@ -1,22 +1,22 @@
 # Context Free
 
 ##  Who is this programming language for?
-
++ This language is for people who want to programatically create images. 
```

Comment:
Does the different features of the language change who might want to use it most?